tool/generate/pictogram.js

```diff
@@ -30,4 +31,42 @@ const generatePictogram = async (dir) => {
   createFile(jsonPath, JSON.stringify(json));
 };
 
-module.exports = { generatePictogram };
+const generateNewPictogram = async (dir) => {
+  _parseFolder(dir);
+};
+
+const _parseFolder = (mainPath, subPath = '') => {
+  const dir = `${mainPath}/${subPath}`;
```

Comment:
```suggestion
  const dir = `${mainPath}${subPath}`;
```
la variable `subPath` commence par un `/` => `dir` est de type `./src/core/asset/artwork//pictograms`